listener/avalon_listener/tcs_work_order_handler.py

```diff
@@ -131,14 +131,18 @@ def WorkOrderGetResult(self, **params):
                 err_code = response["error"]["code"]
                 err_msg = response["error"]["message"]
 
+                data = {
```

Comment:
This is taken care in PR#367

Reply:
I shall remove this once PR#367 is merged.